client/components/jetpack/jetpack-product-info/index.tsx

```diff
@@ -42,10 +43,18 @@ const JetpackProductInfo: FunctionComponent< JetpackProductInfoProps > = ( {
 			{ full && recommendedFor && <JetpackProductInfoRecommendationTags tags={ recommendedFor } /> }
 
 			{ product.productsIncluded ? (
-				<JetpackProductInfoProductList
-					products={ product.productsIncluded }
-					descriptionMap={ descriptionMap }
-				/>
+				<>
+					<JetpackProductInfoProductList
+						products={ product.productsIncluded }
+						descriptionMap={ descriptionMap }
+					/>
+
+					{ alsoIncluded?.length && (
+						<JetpackProductInfoSection title={ translate( 'Also included:' ) }>
```

Comment:
I'm curious about this one.

Will `translate( 'Also included:' )` be translated considering the fact the translations are done for this PR?

Reply:
This one should already be translated. I just copied this string from the old lightbox. 👍